sdk/storage/azure-storage-blob/apilisting.txt

```diff
@@ -0,0 +1,591 @@
+	@classmethod
+	def deserialize(cls, data: str, content_type: str=None): ...
+	@classmethod
+	def enable_additional_properties_sending(cls): ...
+	@classmethod
+	def from_dict(cls, data: dict, key_extractors=None, content_type: str=None): ...
+	@classmethod
+	def is_xml_model(cls): ...
+	def serialize(self, keep_readonly: bool=False) -> dict: ...
+	def validate(self) -> list: ...
+
+class AccountPermissions(object):
+	AccountPermissions.ADD:AccountPermissions
+	AccountPermissions.CREATE:AccountPermissions
+	AccountPermissions.DELETE:AccountPermissions
+	AccountPermissions.LIST:AccountPermissions
+	AccountPermissions.PROCESS:AccountPermissions
+	AccountPermissions.READ:AccountPermissions
+	AccountPermissions.UPDATE:AccountPermissions
+	AccountPermissions.WRITE:AccountPermissions
+	read:bool
+	write:bool
+	delete:bool
+	list:bool
+	add:bool
+	create:bool
+	update:bool
+	process:bool
+	_str:str
+
+	def __add__(self, other): ...
+	def __init__(self, read: bool=False, write: bool=False, delete: bool=False, list: bool=False, add: bool=False, create: bool=False, update: bool=False, process: bool=False, _str: str=None, *, AccountPermissions.ADD, AccountPermissions.CREATE, AccountPermissions.DELETE, AccountPermissions.LIST, AccountPermissions.PROCESS, AccountPermissions.READ, AccountPermissions.UPDATE, AccountPermissions.WRITE): ...
+	def __or__(self, other): ...
+	def __str__(self): ...
+
+class BlobBlock(DictMixin, object):
+	block_id:str
+	state:str
+	size:int
+
+	def __init__(self, block_id: str=None, state: str=Latest, *, size): ...
+	def __str__(self): ...
+	def get(self, key, default=None): ...
+	def has_key(self, k): ...
+	def items(self): ...
+	def keys(self): ...
+	def update(self, *args, **kwargs): ...
+	def values(self): ...
+
+class BlobClient(AsyncStorageAccountHostsMixin, StorageAccountHostsMixin, object):
+	url:str
+	primary_endpoint:str
+	primary_hostname:str
+	secondary_endpoint:str
+	secondary_hostname:str
+	location_mode:str
+	blob_url:str
+	container:container
+	blob:blob
+	snapshot:str
+	credential
+
+	def __aenter__(self): ...
+	def __aexit__(self, *args): ...
+	def __enter__(self): ...
+	def __exit__(self, *args): ...
+	def __init__(self, blob_url: str, container: container=None, blob: blob=None, snapshot: str=None, credential=None, loop=None, *, url, primary_endpoint, primary_hostname, secondary_endpoint, secondary_hostname, location_mode, **kwargs): ...
+	def abort_copy(self, copy_id: copy_id, **kwargs) -> None: ...
+	def acquire_lease(self, lease_duration: int=-1, lease_id: str=None, *, if_modified_since, if_unmodified_since, if_match, if_none_match, timeout, **kwargs) -> ~azure.storage.blob.aio.lease_async.LeaseClient: ...
+	def append_block(self, data, length: int=None, validate_content: bool=False, maxsize_condition: int=None, appendpos_condition: int=None, *, lease, if_modified_since, if_unmodified_since, if_match, if_none_match, encoding, cpk, timeout, **kwargs) -> dict(str, Any): ...
```

Comment:
`appendpos_condition` should be `**kwonly`

Reply:
maxsize_condition can be **kwonly